engine/tests/transport/utils/column_factory.cu

```diff
@@ -0,0 +1,93 @@
+#include "column_factory.h"
```

Comment:
Delete this, use already established way of creating test data

Reply:
ok!